samples/cpp/benchmark_app/main.cpp

```diff
@@ -988,60 +1006,21 @@ int main(int argc, char* argv[]) {
                 }
                 statistics->add_parameters(
                     StatisticsReport::Category::EXECUTION_RESULTS,
-                    {
-                        {latency_label, double_to_string(generalLatency.percentile(FLAGS_latency_percentile))},
-                    });
-                statistics->add_parameters(StatisticsReport::Category::EXECUTION_RESULTS,
-                                           {
-                                               {"Average latency (ms)", double_to_string(generalLatency.average())},
-                                           });
-                statistics->add_parameters(StatisticsReport::Category::EXECUTION_RESULTS,
-                                           {
-                                               {"Min latency (ms)", double_to_string(generalLatency.min())},
-                                           });
-                statistics->add_parameters(StatisticsReport::Category::EXECUTION_RESULTS,
-                                           {
-                                               {"Max latency (ms)", double_to_string(generalLatency.max())},
-                                           });
+                    {StatisticsVariant(latency_label, "latency_median", generalLatency.median_or_percentile),
```

Comment:
```suggestion
                    {StatisticsVariant(latency_label, latency_label, generalLatency.median_or_percentile),
```
label here for json report also should depend on percentile, shouldn't it?

Reply:
It's better keep labels in JSON standard, avoiding changing their names. I've added "percentile_boundary" field to JSON to check if it's truly median or percentile.

Reply:
maybe I should rename it to "median_boundary"  ?

Reply:
percentile_boundary is ok, fits the meaning.